registries/multiagent_decomposer.hocon

```diff
@@ -30,15 +30,26 @@ I can take a problem and solve it by breaking it up into smaller pieces.
                             "type": "string",
                             "description": "Description of a problem."
                         },
-                    },
-                    "required": []
+                        "max_depth": {
+                            "type": "int",
+                            "description": "The maximum depth of solution decomposition."
+                            "default": 5
+                        },
+                        "winning_vote_count": {
+                            "type": "int",
+                            "description": "Number of votes required to win."
+                            "default": 2
+                        }
```

Comment:
Allow for max_depth and winning_vote_count to be passed into the agent.
It's possible we might need more for consistency with winning_vote_count-derived params.
Well see.